R/derive_merged.R

```diff
@@ -1038,3 +1038,170 @@ derive_vars_merged_lookup <- function(dataset,
 get_not_mapped <- function() {
   .temp$nmap
 }
+
+#' Merge a Summary Variable
```

Comment:
What about User-Defined Summary Variable??  Or is this kind of implicit anyways...???

Reply:
I guess everything is a bit user defined, but we let them put in functions so was just musing...